.github/workflows/build_image.yml

```diff
@@ -0,0 +1,26 @@
+name: Build docker image
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+  
+jobs:
+
+    build-docker:
+        runs-on: ubuntu-latest
+        steps:
+        - uses: actions/checkout@v2
```

Comment:
why use deprecated build-push-action@v1, you probably need to use v2
you will have to modify the action inputs a bit, `path` -> `context`, and also don't forget to add `push` input as it defaults to false in the newer versions.

Reply:
changed to v2 https://github.com/threefoldtech/threefold_hub/pull/70/commits/9e3bac9ad19c8ddf8787e532b633a37255556b06